stats/src/main/java/com/facebook/stats/mx/JVMStatsExporter.java

```diff
@@ -49,6 +54,34 @@
     .add(SimpleType.DOUBLE)
     .build();
 
+  private static final Set<String> NUMERIC_STRING_TYPES = ImmutableSet.<String>builder()
+    .add(byte.class.getName()).add(Byte.class.getName())
+    .add(short.class.getName()).add(Short.class.getName())
+    .add(int.class.getName()).add(Integer.class.getName())
+    .add(long.class.getName()).add(Long.class.getName())
+    .add(float.class.getName()).add(Float.class.getName())
+    .add(double.class.getName()).add(Double.class.getName())
+    .build();
+
+  /**
+   * Decides whether to include MBean attributes in the Stats object, and, if
+   * so, what to call them.
+   */
+  public interface StatsNameBuilder {
```

Comment:
Per the [Code Standards](https://github.com/facebook/jcommon/wiki/Coding-Standards#order-of-components-in-a-class), declarations of nested inner classes/interfaces should be after everything else in the class.